packages/nc-gui-v2/composables/useViewData.ts

```diff
@@ -72,7 +76,13 @@ export default (meta: Ref<TableType> | ComputedRef<TableType> | undefined) => {
       return o
     }, {})
 
-    const insertedData = await $api.dbTableRow.create('noco', project?.value.id as string, meta?.value.id as string, insertObj)
+    const insertedData = await $api.dbViewRow.create(
+      'noco',
```

Comment:
can we export the name as a const somewhere?

maybe in lib/constants or something, so we don't have to use the 'noco' string repeadetly.
Maybe something like
```ts
export const NocoName = 'noco' as const 
```

(as const forces the string to be 'noco' not 'string', only for typescript though)

Reply:
make sense I will keep a file